Formula/freetype.rb

```diff
@@ -34,6 +34,10 @@ def install
     system "./configure", "--prefix=#{prefix}", "--without-harfbuzz"
     system "make"
     system "make", "install"
+
+    %W[#{bin}/freetype-config #{lib}/pkgconfig/freetype2.pc].each do |f|
+      inreplace f, prefix, opt_prefix
+    end
```

Comment:
The second is my preference since we have the replacement pair on the second line all the time, and there's no reason to avoid it.